app/models/assignment_form.rb

```diff
@@ -222,10 +223,21 @@ def add_to_delayed_queue
   end
 
   # Find an AQ based on the given values
-  def assignment_questionnaire(questionnaire_type, round_number, topic_id)
+  def assignment_questionnaire(questionnaire_type, round_number, topic_id, duty_id=nil)
```

Comment:
My motivation to use nil was to provide a default argument to the function so that I don't have to change all the places where it has been implemented. 
I plan to refactor this function so that logic for topic-based reviewing and role-based reviewing to make it more clean